lib/matplotlib/tests/test_tightlayout.py

```diff
@@ -157,6 +158,15 @@ def test_tight_layout8():
     example_plot(ax, fontsize=24)
 
 
+@image_comparison(baseline_images=['tight_layout9'])
+def test_tight_layout9():
```

Comment:
Bear with me here, I have no idea how `matplotlib` handles its tests. :sweat_smile: Was planning on getting back to this in a later commit, but you've caught me red-handed.